.github/workflows/linter.yml

```diff
@@ -0,0 +1,19 @@
+name: flake8 linter
+
+on: [push, pull_request]
+
+jobs:
+  flake8:
+    runs-on: ubuntu-latest
+    steps:
+        - uses: actions/checkout@v3
+        - name: Set up Python
+          uses: actions/setup-python@v4
+          with:
+            python-version: '3.9.12'
```

Comment:
Is there a reason why this is not the same python version we're using for the Docker image?
Should not make any difference I guess, but looks super specific here

Reply:
Indeed, [here](https://github.com/uhh-pd-ml/sk_cathode/actions/runs/11343103320) I observed a failure because the Ubuntu version in the GitHub machine is not compatible with our default python version:
```
The version '3.9.7' with architecture 'x64' was not found for Ubuntu 22.04.
```
As a lazy workaround, I picked the next higher version listed in the post [here](https://github.com/actions/setup-python/issues/561#issuecomment-1340657956).

Reply:
Aah I see, sounds good!